### PR TITLE
progression.md: Minor edits to fix a few small grammatical errors

### PR DESCRIPTION
**Pull request by nywilken**

### CODE REVIEW

File: about/conception/progression.md [hunk 1]
```diff
@@ -30,11 +30,11 @@ Typically you've completed an RPG when you've completed all of the core quests.
...
 
-The core exercises give learners a clear path forward. The end is in sight.
+The core exercises give learners a clear path forward; The end is in sight.
```
Comment (rpottsoh) [inline about/conception/progression.md]: `The end is in sight` the `T` in the `The` should be lower case.

File: about/conception/progression.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
+A feeling of progression is key in nearly all human endeavour, but especially when investing in education. You must feel that there is some end in sight and that you are moving towards it. In education, this can be hard, especially in subjects in which practice is a core tenet. Think back to school when you were doing math homework and had to wade through hundreds of examples in algebra. While many of us probably took some perverse delight in this, for most students this sort of practice is painful and disheartening because it is hard to gauge progress. Variety is in short supply and boredom easily sets in.
```
Comment (kytrinyx) [inline about/conception/progression.md:3]: _Maths_ is British English, whereas _math_ is American English. I don't know that we've specified which spelling we prefer. I do suspect that most of our documentation uses American English, though, so I am OK to merge this.

**Review by rpottsoh — CHANGES_REQUESTED** [review]

**Comment by rpottsoh** [comment]
@nywilken thanks for taking the time to pass such a fine toothed comb over this document.

**Review by kytrinyx — APPROVED** [review]
This looks good to me. Thank you, @nywilken

File: about/conception/progression.md (hunk 1 above)
Comment (kytrinyx) [reply about/conception/progression.md]: ah, nice catch.

Comment (nywilken) [reply about/conception/progression.md]: 👍 updated.

File: about/conception/progression.md (hunk 2 above)
Comment (nywilken) [reply about/conception/progression.md:3]: Ah, good to know. Thanks for reviewing.

**Pull request merged**